### DISCUSSION TITLE
Please add user define button for callsign lookup

### DISCUSSION THREAD

**Discussion by liathit** [foldynl/QLog#229] [Ideas]
Please add user define button for callsign lookup. Cqrlog have that feature that very comfortably

![Screenshot from 2023-07-24 09-06-16](https://github.com/foldynl/QLog/assets/1136932/cfd11145-4eb7-49ab-b962-f1e453e3a377)

![Screenshot from 2023-07-24 09-07-08](https://github.com/foldynl/QLog/assets/1136932/ca83ef3f-25da-4da9-bb89-e19f846b0a31)

**Comment by foldynl** [comment]
interesting. I will think about it.
